Update test your skills images and forms tasks to use live samples (#42675)
mdn/content: files/en-us/learn_web_development/core/styling_basics/test_your_skills/images/index.md

In this task, you have an image that is overflowing the box. We want the image to scale down to fit inside the box without any extra white space, but we do not mind if some part of the image is cropped. Update the CSS to achieve this.

Your final result should look like the following rendering:

> {{EmbedLiveSample("images-forms1-finish", "", "400px")}}

Author: done

## Task 3

> Our second form styling assessment is fairly free-form, and you have a lot of flexibility over what you end up doing here, hence we won't provide an example rendering.

Reviewer: third?
or just "This assessment..."?
Can also drop the first "form" because it is in close proximity to "free-form"
```suggestion
The solution for this assessment is fairly free-form, and you have a lot of flexibility over what you can do here. So we're not providing an example rendering.
```
